packages/http-client-csharp/generator/TestProjects/Spector.Tests/Http/_Type/Property/AdditionalProperties/AdditionalPropertiesTests.cs

```diff
@@ -777,39 +777,6 @@ public Task SpreadModelArrayPut() => Test(async host =>
             Assert.AreEqual(204, response.GetRawResponse().Status);
         });
 
-        [SpectorTest]
-        public Task SpreadRecordDiscriminatedUnionGet() => Test(async host =>
-        {
-            var response = await new AdditionalPropertiesClient(host, null).GetSpreadRecordDiscriminatedUnionClient().GetAsync();
-            Assert.AreEqual(200, response.GetRawResponse().Status);
-            Assert.AreEqual("abc", response.Value.Name);
-            Assert.AreEqual(2, response.Value.AdditionalProperties.Count);
-            var prop1 = ModelReaderWriter.Read<WidgetData0>(response.Value.AdditionalProperties["prop1"]);
-            Assert.AreEqual("abc", prop1!.FooProp);
-            var prop2 = ModelReaderWriter.Read<WidgetData1>(response.Value.AdditionalProperties["prop2"]);
-            Assert.AreEqual(new DateTimeOffset(2021, 1, 1, 0, 0, 0, TimeSpan.Zero), prop2!.Start);
-            Assert.AreEqual(new DateTimeOffset(2021, 1, 2, 0, 0, 0, TimeSpan.Zero), prop2.End);
-        });
-
-        [SpectorTest]
-        [Ignore("https://github.com/microsoft/typespec/issues/6426")]
-        public Task SpreadRecordDiscriminatedUnionPut() => Test(async host =>
```

Comment:
Why are we deleting vs marking as Ignore?

Reply:
Same here.